packages/create-react-native-library/src/exampleApp/generateExampleApp.ts

```diff
@@ -231,15 +228,16 @@ export default async function generateExampleApp({
     const app = await fs.readJSON(path.join(directory, 'app.json'));
 
     app.expo.android = app.expo.android || {};
-    app.expo.android.package = `${project.package}.example`;
+    app.expo.android.package = `${config.project.package}.example`;
     app.expo.ios = app.expo.ios || {};
-    app.expo.ios.bundleIdentifier = `${project.package}.example`;
+    app.expo.ios.bundleIdentifier = `${config.project.package}.example`;
 
     await fs.writeJSON(path.join(directory, 'app.json'), app, {
       spaces: 2,
     });
   }
 
+  // Sort the deps by name
```

Comment:
Isn't it evident by the name of `sortObjectKeys` function?

For comment it'd be better to say why 

```suggestion
  // Sort the deps by name to match behavior of package managers
  // This way the package.json doesn't get updated when installing deps
```